test/performance/thread_safe_jit.cpp

```diff
@@ -12,11 +12,15 @@
 using namespace Halide;
 using namespace Halide::Tools;
 
+// The Halide compiler is currently not guaranteed to be thread safe.
```

Comment:
It's not? I thought we fixed all the non-threadsafe stuff a long time ago.

Reply:
Should be fine if no Halide objects are shared across threads. I.e. we don't have any non-threadsafe hidden global state (though temporaries will get non-deterministic names due to atomic counters used to name them).

But yeah the individual objects may have non-threadsafe state, so if you share Halide objects across threads and mutate them bad things will happen.